### DISCUSSION TITLE
在选中单词的右键菜单内添加“快速增加术语”

### DISCUSSION THREAD

**Discussion by Hedgehog-T** [immersive-translate/immersive-translate#2310] [Ideas]
### 新功能 | New Feature

1. 希望新增功能：在原文某一单词上右键，弹出右键菜单中，可以快速将选中单词加入术语表
2. （可选）其实更想要术语表可以支持正则。以生物医药为例子，很多新的单词是满足构词法的，比如“词尾为-mab且长度超过8个字母”一般就是药物名。

### 为什么您想要添加这些功能 | Why you want to add these features

1.背景：每次新增加术语需要手动点开配置并输入，较为繁琐。且各领域经常出现新的名词、新的缩写、甚至页面内的局部缩写，添加起来更繁琐了。

### 提议 | Proposal

已有基础及参考：
    1. 本插件已有右键菜单中的“翻译为简体中文”
    2. 选中单词后传递文本参数具有可行性，例如常规的复制、Edge自带的“在Web中检索这个词”、同为插件的“ACG Helper”的右键“在bilibili中搜索”

**Comment by Hedgehog-T** [comment]
对于术语表部分的正则请求，这里进行补充。以 https://github.com/immersive-translate/immersive-translate/discussions/2161#discussion-7293962 为例子，存在一定的阅读需求是“针对全大写单词（[A-Z]）不进行翻译”，如果能增加正则，此类需求也可以对应解决。

**Comment by liruifengv** [comment]
最新版新增了术语表功能，可以在 Release 下载试用一下，给一些反馈
